hack/virtc/main.go

```diff
@@ -227,7 +228,14 @@ func buildPodConfig(context *cli.Context) (vc.PodConfig, error) {
 		Memory: vmMemory,
 	}
 
+	id := context.String("pod-id")
```

Comment:
@sameo according to your comment, the generation of the UUID in virtc done in this PR is not valid, right ?